automation/test.sh

```diff
@@ -200,7 +210,7 @@ done
 set -e
 make bazel-build-images
 
-trap '{ echo "Dump kubevirt state:"; make dump; }' ERR
+trap '{ collect_debug_logs; echo "Dump kubevirt state:"; make dump; }' ERR
```

Comment:
would you add exit 1 in the end please?
because as we groomed, it suppress the -e
we can do it on other PR if needed, up to you
thanks

Reply:
I could not recreate this, the exit trap is executed after the error trap.
```
#!/bin/bash -e                 
                           
echo "Starting"                
                               
trap '{ echo Exit-trap; }' EXIT
                               
trap '{ echo Err-trap; }' ERR  
false                          
trap - ERR                     
                               
echo MormExit                  
```

Reply:
ah didnt see there is another trap EXIT before
thanks